source/main.ts

```diff
@@ -109,6 +116,17 @@ app.whenReady().then(() => {
     console.error(err)
     app.exit(1)
   })
+
+  tray = new Tray(path.join(__dirname, 'assets/icons/icon.ico'))
+  tray.on('click', () => zettlr?.getMainWindow()?.show())
```

Comment:
When #15 and #37 is complete the tray code should be `source/main/modules/window-manager/index.ts`. Then this line should be:
```typescript
  this._tray.on('click', () => this.showAnyWindow())
```